.github/workflows/tilt.yaml

```diff
@@ -6,32 +6,20 @@ on:
     - 'api/**'
     - 'cmd/**'
     - 'config/**'
+    - 'catalogd/**'
     - 'internal/**'
     - 'pkg/**'
     - 'Tiltfile'
+    - '.tilt-support'
   merge_group:
 
 jobs:
   tilt:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
-      with:
-        repository: operator-framework/tilt-support
-        path: tilt-support
     - uses: actions/checkout@v4
       with:
         path: operator-controller
-    - name: Get catalogd version
-      id: get-catalogd-version
-      run: |
-        cd operator-controller
-        echo "CATALOGD_VERSION=$(go list -mod=mod -m -f "{{.Version}}" github.com/operator-framework/catalogd)" >> "$GITHUB_OUTPUT"
-    - uses: actions/checkout@v4
-      with:
-        repository: operator-framework/catalogd
-        path: catalogd
-        ref: "${{ steps.get-catalogd-version.outputs.CATALOGD_VERSION }}"
     - name: Install Go
       uses: actions/setup-go@v5
       with:
```

Comment:
Hi @joelanford 

after your fix for tilt and now that we are using the code changes in the repo to call in the action, see:
I think we no longer need to have a config inside of catalogd and call it alone, so I removed: https://github.com/operator-framework/operator-controller/pull/1542/commits/b8c8e0c6b23c19b00258d780c3ef6aff09ecc2eb

Let me know if I misunderstood it.